scripts/utils/file-path/url/index.test.mjs

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import { describe, it, expect, vi } from 'vitest'
+import { getUrlFromFilePath } from './index.mjs'
+import { getProductDirectoryFromFilePath } from '../product-directory/index.mjs'
+import { getVersionFromFilePath } from '../version/index.mjs'
+import { PRODUCT_CONFIG } from '../../../../__fixtures__/productConfig.mjs'
+import allDocsPathsJsonMock from '../../../../__fixtures__/docsPathsAllVersionsMock.json'
+
+vi.mock('../product-directory/index.mjs', () => {
```

Comment:
Good job adding tests. ❤️ 

Can you test that these tests run when prebuild isn't run? So when:
`app/api/versionMetadata.json` & `app/api/docsPathsAllVersions.json` are missing. This will tell us if you are mocking everything correctly.